Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragment.kt

```diff
@@ -177,4 +180,25 @@ class HomeFragment : Fragment(R.layout.fragment_home), AutoInject {
             setOnClickListener { homeMenu.showMenuFor(it) }
         }
     }
+
+    private fun showRiskLevelLoweredDialogIfNeeded() {
```

Comment:
The viewmodel should decide whether the dialog needs to be shown, preferably via `SingleLiveEvent`

Reply:
Ok, the showing of the notification is now handled by the viewmodel.

Reply:
The logic and decisions are still being made in the Fragment and not the ViewModel, it would also be good to not require a manual refresh call. Maybe we can have call.